code/game/objects/structures/ai_core.dm

```diff
@@ -313,7 +388,7 @@ That prevents a few funky behaviors.
 	return TRUE
 
 /obj/structure/ai_core/transfer_ai(interaction, mob/user, mob/living/silicon/ai/AI, obj/item/aicard/card)
-	if(state != AI_READY_CORE || !..())
+	if(state != AI_READY_CORE || !..() || brain?.brainmob?.mind || brain?.brainmob?.suiciding)
```

Comment:
@MrMelbert it is supposed to be `mind`, I changed AI core construction to always require a [non suicided] brain (even from monkeys)
We don't want to do "AI intellicard swipe on empty core" if that core's MMI has a mind (which it would if you reversed construction by deconstructing an AI)